lib/matobjplist.gi

```diff
@@ -639,120 +641,120 @@ InstallMethod( Matrix, "for a list and a plist matrix",
+#   end );
+# 
+# InstallMethod( Add, "for a plist matrix and a plist vector",
+#   [ IsPlistMatrixRep and IsMutable, IsPlistVectorRep ],
+#   function( m, v )
+#     Add(m![ROWSPOS],v);
+#   end );
+# 
+# InstallMethod( Add, "for a plist matrix, a plist vector, and a pos. int",
+#   [ IsPlistMatrixRep and IsMutable, IsPlistVectorRep, IsPosInt ],
+#   function( m, v, p )
+#     Add(m![ROWSPOS],v,p);
+#   end );
+# 
+# InstallMethod( Remove, "for a plist matrix",
+#   [ IsPlistMatrixRep and IsMutable ],
+#   function( m )
+#     Remove( m![ROWSPOS] );
+#   end );
+# 
+# InstallMethod( Remove, "for a plist matrix, and a position",
+#   [ IsPlistMatrixRep and IsMutable, IsPosInt ],
+#   function( m, p )
+#     Remove( m![ROWSPOS],p );
+#   end );
+# 
+# InstallMethod( IsBound\[\], "for a plist matrix, and a position",
+#   [ IsPlistMatrixRep, IsPosInt ],
+#   function( m, p )
+#     return p <= Length(m![ROWSPOS]);
+#   end );
+# 
+# InstallMethod( Unbind\[\], "for a plist matrix, and a position",
+#   [ IsPlistMatrixRep and IsMutable, IsPosInt ],
+#   function( m, p )
+#     if p <> Length(m![ROWSPOS]) then
+#         ErrorNoReturn("Unbind\\[\\]: Matrices must stay dense, you cannot Unbind here");
+#     fi;
+#     Unbind( m![ROWSPOS][p] );
+#   end );
+# 
+# InstallMethod( \{\}\:\=, "for a plist matrix, a list, and a plist matrix",
+#   [ IsPlistMatrixRep and IsMutable, IsList,
+#     IsPlistMatrixRep ],
+#   function( m, pp, n )
+#     m![ROWSPOS]{pp} := n![ROWSPOS];
+#   end );
+# 
+# InstallMethod( Append, "for two plist matrices",
+#   [ IsPlistMatrixRep and IsMutable, IsPlistMatrixRep ],
+#   function( m, n )
+#     Append(m![ROWSPOS],n![ROWSPOS]);
+#   end );
+# 
+# InstallMethod( ShallowCopy, "for a plist matrix",
+#   [ IsPlistMatrixRep ],
+#   function( m )
+#     local res;
+#     res := Objectify(TypeObj(m),[m![BDPOS],m![EMPOS],m![RLPOS],
+#                                  ShallowCopy(m![ROWSPOS])]);
+#     if not IsMutable(m) then
+#         SetFilterObj(res,IsMutable);
+#     fi;
+#     return res;
+#   end );
+# 
 InstallMethod( PostMakeImmutable, "for a plist matrix",
   [ IsPlistMatrixRep ],
   function( m )
     MakeImmutable( m![ROWSPOS] );
   end );
 
-InstallMethod( ListOp, "for a plist matrix",
-  [ IsPlistMatrixRep ],
-  function( m )
-    return List(m![ROWSPOS]);
-  end );
-
-InstallMethod( ListOp, "for a plist matrix and a function",
-  [ IsPlistMatrixRep, IsFunction ],
-  function( m, f )
-    return List(m![ROWSPOS],f);
-  end );
-
-InstallMethod( Unpack, "for a plist matrix",
-  [ IsPlistMatrixRep ],
-  function( m )
-    return List(m![ROWSPOS],v->ShallowCopy(v![ELSPOS]));
-  end );
+# InstallMethod( ListOp, "for a plist matrix",
+#   [ IsPlistMatrixRep ],
+#   function( m )
+#     return List(m![ROWSPOS]);
+#   end );
+# 
+# InstallMethod( ListOp, "for a plist matrix and a function",
+#   [ IsPlistMatrixRep, IsFunction ],
+#   function( m, f )
+#     return List(m![ROWSPOS],f);
+#   end );
```

Comment:
MatrixObj are not lists, so remove these. If one wants to extract the rows or columns as VectorObj, then we should add `GetColumnVectors` / `GetRowVectors` operations (better names pending...)